scripts/release_notes/release_notes.ts

```diff
@@ -303,8 +305,19 @@ async function generateNotes(repositories: util.Repo[]) {
   });
 
   // Ask for github token.
-  const token = await util.question(
-      'Enter GitHub token (https://github.com/settings/tokens): ');
+  //const token = await util.question(
+    //  'Enter GitHub token (https://github.com/settings/tokens): ');
+
+ // Getting the github token.
+ const homeDir = process.env.HOME || os.homedir();
+ console.log(homeDir); 
+ const githubTokenPath = process.env.HOME + '/.bashrc';
+ const token = fs.readFileSync(githubTokenPath, 'utf8')
+  .split('\n')
+  .find(line => line.startsWith('export GITHUB_TOKEN='))
+  .split('=')[1]
+  .trim();
```

Comment:
Instead of reading directly from the `.bashrc`, the variable defined in the `.bashrc` should be available in node as `process.env.GITHUB_TOKEN`.